app/models/repository.rb

```diff
@@ -165,6 +175,18 @@ def relative_path(path)
     path
   end
 
+  def required_disk_storage
+    if scm.storage_available?
+      if storage_updated_at.nil? ||
+         storage_updated_at < 12.hours.ago
+
+        Delayed::Job.enqueue ::Scm::StorageUpdaterJob.new(self)
+      end
+
+      required_storage_bytes
```

Comment:
Just a note so that we know it, not neccessarily a request for change:

This seems to be the reason why I will see 0 Bytes for the next hour:

1. ask rails cache if it knows required space
2. calculate required space
3. enqueue a delayed job for repo size
4. return currently known repo size (on init this is 0)
5. the delayed job will update the known repo size

Upon requesting this value I will *never* get the latest value, because with 99.99% certainty I get the cached result before a new one is calculated in the delayed job.